### DISCUSSION TITLE
PCI Compliance [question]

### DISCUSSION THREAD

**Discussion by oscyp** [bennypowers/stripe-elements#81] [Q&A] [answered]
Whether using this library breaks the PCI Compliance?

**Comment by bennypowers** [comment, marked as answer]
It does not, since stripe.js opens an iframe to their domain in the DOM position where you place `<stripe-elements>`

For more info, please consult the stripe.js docs

EDIT: this isn't legal advice, I'm not a lawyer, consult your legal council.
